src/main/java/spring/info/controllers/UserController.java

```diff
@@ -0,0 +1,72 @@
+package spring.info.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import spring.UserResponseDto;
+import spring.info.model.User;
+import spring.info.service.UserService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Sergey Klunniy
+ */
+@RestController
+@RequestMapping("/user")
+public class UserController {
+
+    @Autowired
+    private UserService userService;
+
+    @RequestMapping(value = "/get/{id}", method = RequestMethod.GET)
+    public User getUser(@PathVariable Long id) {
```

Comment:
```suggestion
    public UserResponseDto getUser(@PathVariable Long id) {
```
ми не маємо повертати юзера. це небезпечно

Reply:
чому не виправив цей комент?